cmd/clusterawsadm/cloudformation/bootstrap/template.go

```diff
@@ -96,6 +97,16 @@ func (t Template) RenderCloudFormation() *cloudformation.Template {
 		Roles:             t.controllersPolicyRoleAttachments(),
 	}
 
+	if t.Spec.EKS.Enable {
```

Comment:
This field is going to be changed with https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/2648

Let's wait for it to be merged first and then rebase.
/hold